README.md

```diff
@@ -6,7 +6,7 @@
 [![Appveyor](https://ci.appveyor.com/api/projects/status/ane9s9m78ij6i1hq/branch/master?svg=true)](https://ci.appveyor.com/project/epage/ci-detective/branch/master)
 [![Codeship](https://app.codeship.com/projects/d498c1e0-f33e-0136-19a0-4e3e1edea9c6/status?branch=master)](https://app.codeship.com/projects/320593)
 [![CircleCI](https://circleci.com/gh/crate-ci/ci-detective/tree/master.svg?style=svg)](https://circleci.com/gh/crate-ci/ci-detective/tree/master)
-[![Azure Pipelines](https://dev.azure.com/eopage/ci-detective/_apis/build/status/crate-ci.ci-detective?branchName=master)](https://dev.azure.com/eopage/ci-detective/_build/latest?definitionId=1?branchName=master)
+[![Azure Pipelines](https://dev.azure.com/eopage/ci-detective/_apis/build/status/crate-ci.ci-detective?branchName=master)](https://dev.azure.com/eopage/ci-detective/_build/latest?definitionId=1?branchName=master)  
```

Comment:
I bet my text editor will end up deleting this.

What were your thoughts on my proposed solution in #9?